deluge_web_client/client.py

```diff
@@ -78,26 +79,14 @@ def _connect_to_first_host(self, timeout: int) -> dict[str, Response]:
             )
         }
 
-    def close_session(self) -> None:
-        """
-        Closes the `DelugeWebClient` session.
-
-        This is handled automatically
-        when `DelugeWebClient` is used in a context manager.
-        """
-        self.session.close()
-
```

Comment:
Should we still provide a way for the user to close the session? I noticed you deleted this but didn't see why in your comment.

Reply:
that wasnt intentional, you pushed 1.0.2 while i was editing and i copied the files back over, def should readd. sorry about that, shoulda rebased instead

Reply:
Ahhh, that makes sense!